aws/resource_aws_glue_job.go

```diff
@@ -331,6 +336,7 @@ func expandGlueJobCommand(l []interface{}) *glue.JobCommand {
 	jobCommand := &glue.JobCommand{
 		Name:           aws.String(m["name"].(string)),
 		ScriptLocation: aws.String(m["script_location"].(string)),
+		PythonVersion:  aws.String(m["python_version"].(string)),
```

Comment:
The API will error if `PythonVersion` is not a valid version so we should not send a value if no `"python_version"` was defined in the Terraform configuration. 

```suggestion
}

if v, ok := m["python_version"].(string); ok &&  v != "" {
  jobCommand.PythonVersion = aws.String(v)
 }
```